src/pages/AllGames.jsx

```diff
@@ -28,34 +80,65 @@ const AllGames = () => {
               </select>
             </div>
             <div className="listing-section">
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
-              <ListingCard />
+              {rateFilteredList &&
+                rateFilteredList.map((product) => {
+                  return categoryList.length > 0 ? (
+                    categoryList.includes(
+                      product.categoryName.toUpperCase()
+                    ) && <ListingCard key={product.id} productItem={product} />
+                  ) : (
+                    <ListingCard key={product.id} productItem={product} />
+                  );
+                })}
             </div>
           </main>
           <aside className="listing-aside">
             <div className="filter-container">
               <h3>Filters</h3>
               <div className="divider-sm"></div>
 
+              <div className="slider-content">
+                <label htmlFor="slider">₹{price}</label>
+                <div className="slider-container">
+                  <input
+                    type="range"
+                    name="slider"
+                    min="0"
+                    max={sliderMaxValue}
+                    step="50"
+                    id="slider"
+                    value={price}
+                    onChange={sliderChangeHandler}
+                  />
+                </div>
+              </div>
+              <div className="divider-sm"></div>
+
               <div className="filter-group">
                 <p className="filter-heading">CATEGORY</p>
                 <div className="filter-input-container">
                   <div className="filter-input-group">
-                    <input type="checkbox" name="racing" />
+                    <input
```

Comment:
One suggestion, you can put this input in the label of category, it will look good